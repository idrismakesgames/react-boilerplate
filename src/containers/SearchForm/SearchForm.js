import React, { Component } from 'react';

export default class SearchForm extends Component {
  state = {};

  render() {
    return (
      <div>
        This is the search form placeholder
        <input type="text" className="input-search" />
      </div>
    );
  }
}

// Get local state disabled search formw hen searching.
// ensure to test for this, checking local component state.

// Reset button to clear and search again.

// Get key strokes into redux state and send and.
// get api results and store them and output them into components.

// Create button to see readme component use browser router and test in jest.
