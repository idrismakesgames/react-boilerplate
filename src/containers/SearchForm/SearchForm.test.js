import React from 'react';
import { shallow } from 'enzyme';
import SearchForm from './SearchForm';

describe('Search Form ', () => {
  const searchForm = shallow(<SearchForm />);

  it('renders properly', () => {
    expect(searchForm).toMatchSnapshot();
  });

  it('contains an input to enter a search command', () => {
    expect(searchForm.find('.input-search').exists()).toEqual(true);
  });
});
