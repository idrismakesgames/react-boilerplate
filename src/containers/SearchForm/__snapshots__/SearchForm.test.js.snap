// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Form  renders properly 1`] = `
<div>
  This is the search form placeholder
  <input
    className="input-search"
    type="text"
  />
</div>
`;
